help/include/links-examples.md

```diff
@@ -0,0 +1,13 @@
+### What you type
+
+```
+Named link: [Zulip homepage](zulip.com)
+A URL (links automatically): zulip.com
+Stream link: #**stream name**
+Stream and topic link: #**stream name>topic name**
+Custom linkifier: For example, #2468 can automatically link to an issue in your tracker.
```

Comment:
I think `#2468` should look like a link in the image instead of plain text.

Reply:
Good point! The stream links too, actually.